batch/batch/resource_usage.py

```diff
@@ -82,6 +117,60 @@ def memory_usage_bytes(self) -> Optional[int]:
                 return int(f.read().rstrip())
         return None
 
+    def overlay_storage_usage_bytes(self) -> int:
+        return shutil.disk_usage(self.container_overlay).used
+
+    def io_storage_usage_bytes(self) -> int:
+        if self.io_volume_mount is not None:
+            return shutil.disk_usage(self.io_volume_mount).used
+        return 0
+
+    async def network_bandwidth(self) -> Tuple[Optional[float], Optional[float]]:
+        async with iptables_lock:
```

Comment:
How come you don't need this lock elsewhere in `worker.py` when invoking iptables?

Reply:
I think you solved this with `-w {MAX_IPTABLES_WAIT_SECONDS}` in worker.py. It blocks for 60 seconds waiting for the lock

I can switch to `-w {MAX_IPTABLES_WAIT_SECONDS}` in resource_usage.py. I think having a global iptables lock is more complicated because resource_usage.py is used by the front end as well.

Let me know what you prefer.

Reply:
Ah I see. Let's just stay consistent by using the wait then.